metaflow/cli.py

```diff
@@ -606,11 +694,124 @@ def run(obj,
     write_latest_run_id(obj, runtime.run_id)
     write_run_id(run_id_file, runtime.run_id)
 
+    # import pickle
+    #
+    # with open('graph.pkl', 'wb') as gh:
+    #     pickle.dump(obj.graph, gh)
+
     parameters.set_parameters(obj.flow, kwargs)
     runtime.persist_parameters()
     runtime.execute()
 
 
+@parameters.add_custom_parameters
+@cli.command(help='Run the initial part of the workflow (i.e., save parameters locally).')
```

Comment:
I think I missed the part of the demo that went over this part of the CLI, apologies for that. Looks like really this CLI command is constructing the local runtime to initialize a run (ie. give it an id, parse parameters, etc.) but then it doesn't actually run any steps?

Reply:
nice catch, a mistake in the help comments.  It should explicitly state that this isn't running a step, just `Setting up initial part of the workflow (i.e., persist parameters)`

Reply:
Thank you! Modified this to better reflect what it does.